app/main/helpers/services.py

```diff
@@ -241,7 +241,7 @@ def _is_pricing_type(key):
 
 def _is_type(key, *types):
     """Return True if a given key is one of the provided types"""
-    return new_service_content.get_question(key)['type'] in types
+    return new_service_content.get_question(key).get('type') in types
```

Comment:
`_is_pricing_type` is called from `pricing_question_filter` we could add a more specific check there but I think that this is actually correct. `_is_pricing_type("non_existent_key")` is `False`.